Services/Interfaces/web_interface/__init__.py

```diff
@@ -106,6 +105,14 @@ def update_registered_plugins(plugins):
 import tentacles.Services.Interfaces.web_interface.flask_util
 
 
+def set_web_interface(web_interface_instance):
+    global _WEB_INTERFACE
+    _WEB_INTERFACE = web_interface_instance
+
+
+def get_web_interface():
+    return _WEB_INTERFACE
+
```

Comment:
There is no existing method to get a service instance?

Reply:
For services but not for interfaces. However we could get the instant from the octobot interface producer object if you think it's better

Reply:
I think we should limit this kind of implementation outside of the framework. So yes I think from the interface producer object is better or we could also add this feature to abstract interface tentacle.

Reply:
👍 i'll add it to the octobot api then

Reply:
done